### ISSUE TITLE
Cancel Order page needs vocalized alert that items should be added before proceeding

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13609]
**Describe the bug**
- If no items are added for cancellation an error is displayed on the bottom but no heard.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Add any product to the cart.
3. Checkout the product.
4. Go to Order History and click on the order number.
5. In the Order Details page, scroll down, click "Cancel Items" button.
6. In the Cancel Order Page, press "Continue" button WITHOUT adding any items.
7. Alter at the button is display (hear the absence of vocalization).

**Expected behavior**
- Alert is vocalized by screen reader.
- Use: `aria-live="assertive"` and `aria-atomic="true"`.

**Screenshots**
![Screen Shot 2021-09-02 at 7 19 26 PM](https://user-images.githubusercontent.com/15113219/131929025-e8627fef-f01a-44c5-a2d2-2dbf32fffc82.png)

**Potential solution**

![Screen Shot 2021-09-02 at 7 24 02 PM](https://user-images.githubusercontent.com/15113219/131929218-df0f0e46-600a-4c06-9aed-ef25c9e6ab7a.png)

![Screen Shot 2021-09-02 at 6 53 59 PM](https://user-images.githubusercontent.com/15113219/131929235-6afa9cc8-14c3-4cc5-8bc2-3ae64afa30c9.png)

**Comment by saad-mo** [comment]
moving it to progress since NVDA and JAWS solutions is being researched

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.

For this ticket, it's ok. 
It reads the error message now.
![cancelOrder0](https://user-images.githubusercontent.com/45409515/159776502-7dc27de5-f551-4f8f-a285-491bbf2b88d4.png)


Plus tested the whole cancel process, looks ok also.

![cancelOrder1a](https://user-images.githubusercontent.com/45409515/159776561-6e344e5a-040e-4708-adc5-f260df68dddf.png)
![cancelOrder2a](https://user-images.githubusercontent.com/45409515/159776564-b87275c1-940f-400d-9aea-3f60b163dea3.png)
![cancelorder2](https://user-images.githubusercontent.com/45409515/159776578-5f42f0e8-123e-4722-8988-4529cea7db72.png)
![cancelOrder3](https://user-images.githubusercontent.com/45409515/159776581-f6af94cb-2293-4517-a6d9-9a530c316e47.png)
![cancelOrder4](https://user-images.githubusercontent.com/45409515/159776583-d9cf1edc-994d-471c-a36b-91a90761b86c.png)
![cancelOrder5](https://user-images.githubusercontent.com/45409515/159776585-2ba8633a-fb37-4266-b6ec-077aa6f6cd81.png)
